ingredient/ingredient-metric-alert/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
```

Comment:
Why does your PR have changelog updates? The changelog should only be updated vi the automation in master. Otherwise you're creating NPM versions that are never published

Reply:
Sorry.  Leftover from copying the template ingredient.  I'll remove it from here and also remove it from the template.